WMF Framework/Remote Notifications/Model/RemoteNotification+CoreDataClass.swift

```diff
@@ -15,6 +15,18 @@ import CoreData
     }
 }
 
+public struct RemoteNotificationState: OptionSet {
```

Comment:
could these be modeled as discrete states? it seems like if a notification is excluded, it'll never be seen or read. also, it seems like read implies that it was seen. The states I'm imagining are `ignored` -> `presented` -> `read`